Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.6'
+ruby '~> 2.6.0'
```

Comment:
```suggestion
ruby '~> 2.6'
```

maybe?

Reply:
I would be okay with that -- albeit very loose, but it doesn't mean the same thing.

Reply:
Thanks @SleeplessByte yeah, the proposed would be 2.6 or 2.7 or further, rather than restricting to within 2.6.x?  It has been a little bit since I had to know this.